src/Symfony/Bridge/PhpUnit/README.md

```diff
@@ -0,0 +1,62 @@
+PHPUnit Bridge
+==============
+
+Utilities for PHPUnit, especially user deprecation notices management.
```

Comment:
I'm pretty sure that it was not what I proposed. We have a uniform way to describe each component in one sentence. So, it should be ` Provides utilities ...`